pkg/maglev/maglev.go

```diff
@@ -203,7 +204,7 @@ func GetLookupTable(backendsMap map[string]*loadbalancer.Backend, m uint64) []in
 // MB, multiply by sizeof(uint64).
 func derivePermutationSliceLen(m uint64) uint64 {
 	threshold := uint64(8 * 1024 * 1024 * 1024) // 8GB
-	if vm, err := mem.VirtualMemory(); err != nil || vm == nil || vm.Total <= threshold {
+	if vm, err := memory.Get(); err != nil || vm == nil || vm.Total <= threshold {
```

Comment:
Both libraries retrieve MemTotal from `proc/meminfo`. So the result will be the same.

gopsutil
https://github.com/shirou/gopsutil/blob/v3.23.12/mem/mem_linux.go#L59

go-osstat
https://github.com/mackerelio/go-osstat/blob/v0.2.4/memory/memory_linux.go#L34